src/ScottBrady.IdentityModel/Crypto/ExtendedJsonWebAlgorithmsKeyTypes.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ScottBrady.IdentityModel.Crypto
+{
+    public static class ExtendedJsonWebAlgorithmsKeyTypes
+    {
+        // https://datatracker.ietf.org/doc/html/draft-ietf-jose-cfrg-curves-06#section-2
+        public const string ECDH = "OKP";
```

Comment:
I'll move this one into the `ExtendedSecurityAlgorithms` class, but otherwise all good 🙂